app/components/CrashDiagramCard.tsx

```diff
@@ -2,63 +2,165 @@ import { useState } from "react";
 import Alert from "react-bootstrap/Alert";
 import Card from "react-bootstrap/Card";
 import Image from "react-bootstrap/Image";
+import Button from "react-bootstrap/Button";
+import ButtonGroup from "react-bootstrap/ButtonGroup";
+import { BsZoomIn, BsZoomOut } from "react-icons/bs";
```

Comment:
I added `react-icons` which has bootstrap and a jillion other ones available. I like how you import them here, so you only get the overhead of the buttons you want and not the whole menagerie.

Reply:
yes! great package—thanks for bumping us up a version ;)

For the time being, would you mind switching to the `fa6` icons? I kinda want to be consistent with font awesome v6 until proven otherwise. (Partially because I am considering requesting that we purchase the whole icon set so we can have the much nicer outline and "light" styles).

Here are those imports!

```javascript
import { FaMagnifyingGlassPlus } from "react-icons/fa6";
import { FaMagnifyingGlassMinus } from "react-icons/fa6";
```

Reply:
Oh yea, for sure. Thanks for these. They look good. Do you dig the "undo" button, or do you have a recommendation for that?

Reply:
ah, right...I can't find one that i prefer over what you've got. let's keep it as an exception to the font-awwesome preference 🙏